ironfish-cli/scripts/import-export-test.sh

```diff
@@ -0,0 +1,150 @@
+            }
+            eof
+        }
+    ")
+    # verify return code of import
+    if [ $? -ne 0 ]; then
+        echo "Import failed for $ACCOUNT_NAME"
+        exit 1
+    fi
+    check_import_success "$IMPORT_OUTPUT" "$ACCOUNT_NAME"
+    DELETE_OUTPUT=$(yarn --cwd ./ironfish-cli start wallet:delete $ACCOUNT_NAME --wait)
+    # verify return code of delete
+    if [ $? -ne 0 ]; then
+        echo "Deletion failed for $ACCOUNT_NAME"
+        exit 1
+    fi
+    check_delete_success "$DELETE_OUTPUT" "$ACCOUNT_NAME"
+}
+
+
+function import_account_by_pipe() {
+    echo "Testing import by pipe."
+    IMPORT_OUTPUT=$(expect -c "
+        spawn sh -c \"cat $TEST_FILE | yarn --cwd ./ironfish-cli start wallet:import\"
+        expect {
+            \"Enter a new account name:\" {
+                send \"$ACCOUNT_NAME\\r\"
+                exp_continue
+            }
+            \"Account $ACCOUNT_NAME imported\" {
+                set output \$expect_out(buffer)
+                exp_continue
+            }
+            eof {
+                set output \$expect_out(buffer)
+            }
+        }
+        puts \$output
+    ")
+    # verify return code of import
+    if [ $? -ne 0 ]; then
+        echo "Import failed for $ACCOUNT_NAME"
+        exit 1
+    fi
+    check_import_success "$IMPORT_OUTPUT" "$ACCOUNT_NAME"
+    DELETE_OUTPUT=$(yarn --cwd ./ironfish-cli start wallet:delete $ACCOUNT_NAME --wait)
+    # verify return code of delete
+    if [ $? -ne 0 ]; then
+        echo "Deletion failed for $ACCOUNT_NAME"
+        exit 1
+    fi
+    check_delete_success "$DELETE_OUTPUT" "$ACCOUNT_NAME"
+}
+
+
+
+
+function import_account_by_path() {
+    echo "Testing import by path."
+    # the cwd is already ironfish-cli, so we actually need to trim an extra "ironfish-cli" from the path
+    ACCOUNT_BY_PATH_TEST_FILE="./"${TEST_FILE:14}
+    IMPORT_OUTPUT=$(expect -c "
+        spawn yarn --cwd ./ironfish-cli start wallet:import --path $ACCOUNT_BY_PATH_TEST_FILE
+        expect {
+            \"Enter a new account name:\" {
+                send \"$ACCOUNT_NAME\\r\"
+                exp_continue
+            }
+            \"Account $ACCOUNT_NAME imported\" {
+                set output \$expect_out(buffer)
+            }
+            eof {
+                set output \$expect_out(buffer)
+            }
+        }
+        puts \$output
+    ")
+    # verify return code of import
+    if [ $? -ne 0 ]; then
+        echo "Import failed for $ACCOUNT_NAME"
+        exit 1
+    fi
+    check_import_success "$IMPORT_OUTPUT" "$ACCOUNT_NAME"
+    DELETE_OUTPUT=$(yarn --cwd ./ironfish-cli start wallet:delete $ACCOUNT_NAME --wait)
+    # verify return code of delete
+    if [ $? -ne 0 ]; then
+        echo "Deletion failed for $ACCOUNT_NAME"
+        exit 1
+    fi
+    check_delete_success "$DELETE_OUTPUT" "$ACCOUNT_NAME"
+}
+
+#this script is always run from the root of ironfish
+TEST_VECTOR_LOCATION='./ironfish-cli/scripts/import-export-test-vector/'
```

Comment:
You don't need to depend on this. You can setup bash scripts so it doesn't matter where they are run from. Generally in a bash script the first few lines should be

1. Set error mode to catch errors in the scripts
2. CD to the current directory of the script

By doing that, you never have to care what your CWD is when you execute this script. I ran into this issue when I was just using it.